src/libmusig2.h

```diff
@@ -129,3 +133,15 @@ int musig2_aggregate_partial_sig(secp256k1_context *ctx, musig2_partial_signatur
  *                          : nr_signers: the total number of signers/keys submitted.
  */
 void musig2_prepare_verifier(secp256k1_context *ctx, secp256k1_xonly_pubkey *aggr_pk, secp256k1_pubkey *pk_list, int nr_signers);
+
+
+/** Function    : musig2_signer_precomputation
+ *  Purpose     : Prepares the signer for partial signature generation. Aggregates the public keys and the batch commitments for all messages to be signed.
+ *                Returns 1 if public keys and commitments aggregated successfully, 0 otherwise.
+ *  Parameters  : IN/OUT    : mc: musig2_context objec.
+ *              : IN        : pk_list: list of public keys from all signers
+ *                          : serialized_batch_list: The string including the list of batch commitments of all signers for all states.
```

Comment:
You should specify how the serialisation is expected (compressed vs non-compressed)